packages/apps/graph/src/graph/objects/block.ts

```diff
@@ -86,34 +88,30 @@ export default builder.prismaNode(Prisma.ModelName.Block, {
     }),
 
     // relations
-    transactions: t.prismaConnection({
-      type: Prisma.ModelName.Transaction,
+    transactions: t.field({
```

Comment:
t.connection won't work here I assume right? Due to that totalCount issue?

Reply:
Yep exactly